src/test/scala/io/github/pauljamescleary/petstore/infrastructure/repository/doobie/package.scala

```diff
@@ -1,15 +1,27 @@
 package io.github.pauljamescleary.petstore.infrastructure.repository
 
-import cats.effect.IO
+import cats.implicits._
+import cats.effect.{Async, ContextShift, Effect, IO}
 import io.github.pauljamescleary.petstore.config.{DatabaseConfig, PetStoreConfig}
 import _root_.doobie.Transactor
 
+import scala.concurrent.ExecutionContext
+
 package object doobie {
-  def getTransactor : IO[Transactor[IO]] = for {
-    conf <- PetStoreConfig.load[IO]
-    tr <- DatabaseConfig.dbTransactor[IO](conf.db)
-    x <- DatabaseConfig.initializeDb[IO](conf.db)
-  } yield tr
+  def getTransactor[F[_] : Async : ContextShift](cfg : DatabaseConfig) : Transactor[F] =
```

Comment:
in testing we don't care about performance and mutlithreading really, so there's no reason to use HikariTransactor. Just use default.